codalab/lib/download_manager.py

```diff
@@ -57,6 +58,20 @@ def get_target_info(self, uuid, path, depth):
         bundle_state = self._bundle_model.get_bundle_state(uuid)
         # Raises NotFoundException if uuid is invalid
 
+        def check_dependencies(err):
+            # if path not in bundle, check if it matches one of its dependencies
+            child_path_to_dep = {
+                dep.child_path: dep for dep in self._bundle_model.get_bundle_dependencies(uuid)
+            }
+            matching_dep = child_path_to_dep.get(path.split(os.path.sep)[0], None)
```

Comment:
Hmm but the rest of the `download_manager`/`download_util` code uses `os.path.sep`. I was trying to be consistent with that.
Also I don't even know if this matters since we're running our server on UNIX anyways.